ACPs/236-continuous-staking/README.md

```diff
@@ -0,0 +1,154 @@
+| ACP           | 236                                                         |
+|:--------------|:------------------------------------------------------------|
+| **Title**     | Continuous Staking                                          |
+| **Author(s)** | Razvan Angheluta ([@rrazvan1](https://github.com/rrazvan1)) |
+| **Status**    | Proposed                                                    |
+| **Track**     | Standards                                                   |
+
+## Abstract
+
+This proposal introduces continuous staking for validators on the Avalanche P-Chain. Validators can stake their tokens
+continuously, allowing their stake to compound over time, accruing rewards once per specified cycle.
+
+## Motivation
+
+The current staking system on the Avalanche P-Chain restricts flexibility for stakers, limiting their ability to respond
+to changing market conditions or liquidity needs. Managing a large number of nodes is also challenging, as re-staking at
+the end of each period is labor-intensive, time-consuming, and poses security risks due to
+the required transaction signing. Additionally, tokens can remain idle at the end of a staking period
```

Comment:
Are many users managing a large number of nodes? The staking limit is up to 3M AVAX